article-analyzer/src/main/java/co/inventorsoft/academy/spring/models/Article.java

```diff
@@ -0,0 +1,42 @@
+package co.inventorsoft.academy.spring.models;
+
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.hibernate.annotations.GenericGenerator;
+
+/**
+ * Article model.
+ */
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@ToString
+@Table(name = "articles")
```

Comment:
it is too much, the task is not about storing data in db, remove all of these, should be just pojo